utils/dump_leaderboard.go

```diff
@@ -114,25 +128,43 @@ func main() {
 		members = append(members, member)
 	}
 
+	if *sortFlag == "default" {
+		sort.Sort(ByName(members))
+	} else if *sortFlag == "stars" {
+		sort.Sort(ByStars(members))
+	} else if *sortFlag == "names" {
+		sort.Sort(ByName(members))
+	} else {
+		log.Fatal(fmt.Sprintf("invalid sort: %s. Allowed: %s", *sortFlag, allowedSorts))
+	}
+
 	// Build the completions for the given day.
-	results := map[int][]Result{
-		1: []Result{},
-		2: []Result{},
+	var results [25]map[int][]Result
+
+  // Initialize results map for each day.
```

Comment:
The indent looks wrong. Run gofmt on this file to be sure?

Reply:
I think it has now been formatted with gofmt.